### PR TITLE
Upgrade Webpack + Babel + Dependencies

### PR DESCRIPTION
**Pull request by didierofrivia**
**What this PR does / why we need it**:

We need this upgrade for https://github.com/3scale/porta/pull/761

This needs to be addressed first: https://github.com/3scale/infrastructure/issues/725

### CODE REVIEW

File: bin/webpack [hunk 1]
```diff
@@ -12,4 +12,8 @@ require "bundler/setup"
 
 require "webpacker"
 require "webpacker/webpack_runner"
-Webpacker::WebpackRunner.run(ARGV)
+
+APP_ROOT = File.expand_path("..", __dir__)
+Dir.chdir(APP_ROOT) do
```
Comment (hallelujah) [inline bin/webpack]: This was generated by the webpacker update right?

Comment (didierofrivia) [reply bin/webpack]: yes

File: config/boot.rb [hunk 2]
```diff
@@ -6,6 +6,9 @@
 # just safeguard
 ENV['RAILS_ENV'] = 'production' if ENV['RAILS_ENV'] == 'enterprise'
 
+# ENV needed for webpacker to use npm bin instead of Yarn
+ENV['WEBPACKER_NODE_MODULES_BIN_PATH'] = `npm bin`.chomp
```
Comment (hallelujah) [inline config/boot.rb]: This should not be here because not every task should require npm

Comment (hallelujah) [reply config/boot.rb]: Put it in the webpack bin or when npm is really required

File: .babelrc [hunk 3]
```diff
@@ -1,28 +1,26 @@
...
-}
\ No newline at end of file
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": "entry",
+        "targets": {
+          "node": "current",
+          "ie": "11",
+          "firefox": "62",
```
Comment (damianpm) [inline .babelrc]: We should target Firefox 60, since it's packed with  RHEL 7

Comment (thomasmaas) [reply .babelrc]: https://access.redhat.com/articles/2798521

Comment (thomasmaas) [reply .babelrc]: and this version will be updated with next release: we support the last version at the moment of release. Current version is 67. People can update…

File: .babelrc [hunk 4]
```diff
@@ -1,28 +1,26 @@
...
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": "entry",
+        "targets": {
+          "node": "current",
+          "ie": "11",
+          "firefox": "60",
+          "chrome": "69"
+        }
```
Comment (thomasmaas) [inline .babelrc]: firefox: 67
chrome: 75
EdgeHTML/edge: 18/44
ie: 11

**Comment by damianpm** [comment]
I tested locally, adding PF4/React components, and it works!

File: .babelrc (hunk 4 above)
Comment (damianpm) [reply .babelrc]: Are you sure @thomasmaas ? We had some issues of clients using Firefox 60 (with Service Discovery)

Comment (thomasmaas) [reply .babelrc]: https://access.redhat.com/articles/2798521

which is updated after each release: we support latest.

Comment (damianpm) [reply .babelrc]: OK

**Pull request merged**